tests/20-main.spec.js

```diff
@@ -258,6 +270,111 @@ describe('statusTypeMatches', () => {
 });
 
 describe('checkStatus', () => {
+  it('should verify a valid status list vc', async () => {
+    const credential = {
+      '@context': [
+        'https://www.w3.org/2018/credentials/v1',
+        VC_SL_CONTEXT_URL
+      ],
+      id: 'urn:uuid:a0418a78-7924-11ea-8a23-10bf48838a41',
+      type: ['VerifiableCredential', 'example:TestCredential'],
+      credentialSubject: {
+        id: 'urn:uuid:4886029a-7925-11ea-9274-10bf48838a41',
+        'example:test': 'foo'
+      },
+      credentialStatus: {
+        id: 'https://example.com/status/1#67342',
+        type: 'StatusList2021Entry',
+        statusPurpose: 'revocation',
+        statusListIndex: '67342',
+        statusListCredential: SLC.id
+      },
+      issuer: SLC.issuer,
+    };
+    const suite = new Ed25519Signature2020();
+    const result = await checkStatus({
+      credential,
+      suite,
+      documentLoader,
+      verifyStatusListCredential: true
+    });
+    should.not.exist(result.error);
+    result.verified.should.equal(true);
+  });
+
+  it('should fail to verify an invalid status list vc', async () => {
```

Comment:
We should add another test where the proof isn't deleted, but the SLC is tampered with (changed in some way).

Reply:
Added [292ca49](https://github.com/digitalbazaar/vc-status-list/pull/10/commits/292ca49af0f00b0879d3c990b2778f4a19f5124f)